controllers/accountclaim/accountclaim_controller.go

```diff
@@ -246,6 +246,13 @@ func (r *AccountClaimReconciler) handleAccountClaimDeletion(reqLogger logr.Logge
 		return nil
 	}
 
+  // Workaround for FleetManagers special account handling, see
+  // https://issues.redhat.com/browse/OSD-19093
+  if len(accountClaim.GetFinalizers()) > 1 {
```

Comment:
Should we also check here that when there _is_ only one finalizer that it is also the AAO finalizer? (it wouldn't have to be in this if block, could be a separate block)

Reply:
This is actually already checked as the first statement in this function. So when we reach this part, we are certain that the accountclaim has the AAO finalizer.

Reply:
Ahh, perfect then. That's what I get for not looking at the whole function. Thanks for keeping me honest :D